src/VisualStudio/CSharp/Test/DocumentOutline/DocumentOutlineTestsBase.cs

```diff
@@ -107,7 +107,7 @@ protected async Task<DocumentOutlineTestMocks> CreateMocksAsync(string code)
             async Task<ManualInvocationResponse?> RequestAsync(ITextBuffer textBuffer, Func<JToken, bool> capabilitiesFilter, string languageServerName, string method, Func<ITextSnapshot, JToken> parameterFactory, CancellationToken cancellationToken)
             {
                 var request = parameterFactory(textBuffer.CurrentSnapshot).ToObject<RoslynDocumentSymbolParams>();
-                var response = await testLspServer.ExecuteRequestAsync<RoslynDocumentSymbolParams, DocumentSymbol[]>(method, request!, cancellationToken);
+                var response = await testLspServer.ExecuteRequestAsync<RoslynDocumentSymbolParams, object[]>(method, request!, cancellationToken);
```

Comment:
📝 I lost two hours because this mock isn't the real call and didn't match the signature of the real call:
https://github.com/dotnet/roslyn/blob/1497e87d967c5b7797edb5f782131508607139e5/src/Features/LanguageServer/Protocol/Handler/Symbols/DocumentSymbolsHandler.cs#L41